FWCore/Common/interface/LuminosityBlockBase.h

```diff
@@ -6,9 +6,11 @@
 // Package:     FWCore/Common
 // Class  :     LuminosityBlockBase
 //
-/**\class LuminosityBlockBase LuminosityBlockBase.h FWCore/Common/interface/LuminosityBlockBase.h
+/**\class LuminosityBlockBase LuminosityBlockBase.h
+ FWCore/Common/interface/LuminosityBlockBase.h
 
```

Comment:
I do not know the answer to this question. What does doxygen do if you split a line that is a doxygen command across multiple lines?